src/test/java/org/minions/devfund/marylin/SquarelotronTest.java

```diff
@@ -0,0 +1,583 @@
+package org.minions.devfund.marylin;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+
+/**
+ * SquarelotronTest class.
+ */
+public class SquarelotronTest {
+    /**********************************************************
+     **********************upsideDownFlip**********************
+     *********************************************************/
+
+    /**
+     * {{1, 2}, {3, 4}}.
+     */
+    @Test
+    public void testUpsideDownFlipTwoByTwoGridFirstRing() {
+        final int one = 1;
+        final int two = 2;
+        final int[][] expectedGrid = {{3, 4}, {1, 2}};
+        Squarelotron initialSquarelotronTwoByTwo = new Squarelotron(two);
+        Squarelotron actualSquarelotron = initialSquarelotronTwoByTwo.upsideDownFlip(one);
+        assertArrayEquals("Grid is not the expected.", expectedGrid, actualSquarelotron.getSquarelotron());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Grid is not the expected." 40 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)